pinot-broker/src/main/java/com/linkedin/pinot/broker/broker/helix/HelixBrokerStarter.java

```diff
@@ -188,13 +188,17 @@ public void run() {
     return brokerServerBuilder;
   }
 
-  private String getZkAddressForBroker(String zkServers, String helixClusterName) {
+  public static String getZkAddressForBroker(String zkServers, String helixClusterName) {
     List tokens = new ArrayList<String>();
-
-    for (String token : zkServers.split(",")) {
-      tokens.add(StringUtil.join("/", StringUtils.chomp(token, "/"), helixClusterName, PROPERTY_STORE));
+    String[] zkSplit = zkServers.split("/", 2);
```

Comment:
Can you add a comment that the format should be `127.0.0.1:3000,127.0.0.1:3001,127.0.0.1:3002/app/a` which applies the `/app/a` chroot to all servers and to `@see org.apache.zookeeper.Zookeeper#Zookeeper(String, int, org.apache.zookeeper.Watcher)` for details of the zk connection string.